reactfrontend/src/components/singleAsset/singleAsset.js

```diff
@@ -7,9 +7,26 @@ class SingleAsset extends Component {
         this.state = {
             data: [],
             details: null,
+            asset: {
+              assetTag: '',
+              assetType: '',
+              createdBy: '',
+              assetStatus: '',
+              serialNumber: ''
```

Comment:
Consistent with the 'addAssetForm.js'?

Reply:
doesn't have to be but since it's just naming but it is